package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "3.9.10",
+  "version": "3.10.0",
```

Comment:
This is a "not small" change (and we recently added Registrar's filings, etc) so I updated the minor version,

We don't follow any hard-and-fast rule here but this makes sense to me. Please do the same :)